duplicate-post-admin.php

```diff
@@ -790,14 +776,15 @@ function duplicate_post_copy_post_meta_info( $new_id, $post ) {
 		$meta_blacklist[] = '_thumbnail_id';
 	}
 
+	$meta_blacklist = apply_filters_deprecated( 'duplicate_post_blacklist_filter', array( $meta_blacklist ), '3.2.5', 'duplicate_post_excludelist_filter' );
```

Comment:
Should the version for the deprecation be 4.0.0?

Reply:
The filter was deprecated since version 3.2.5. The line was copied from `master` since this change wasn't ported to the current `develop` branch before.